payrue/skin.scss

```diff
@@ -0,0 +1,9 @@
+.name {
```

Comment:
Is this skin the same as the default one? If yes you can remove it. Otherwise you can change ".name" to ".payrue" here